inifile.py

```diff
@@ -549,8 +549,14 @@ def save(self, create_folder=False):
             except OSError:
                 pass
             reraise(*exc_info)
-
-        os.rename(tmp_filename, self.filename)
+        try:
+            os.rename(tmp_filename, self.filename)
+        except OSError:
+            if os.name == 'nt':
+                os.remove(self.filename)
+                os.rename(tmp_filename, self.filename)
+            else:
+                pass
```

Comment:
make this a raise instead. Also we should probably use `os.replace` if it exists (which is the case on python 3 for instance).  `os.replace` also works that way on windows in some situations.